src/providers/WorkflowCore.Persistence.EntityFramework/WorkflowCore.Persistence.EntityFramework.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <AssemblyTitle>Workflow Core EntityFramework Core Persistence Provider</AssemblyTitle>
     <Authors>DanielGerlag,sergiikram</Authors>
-    <TargetFrameworks>netstandard2.1;net6.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.1;net6.0;net8.0</TargetFrameworks>
```

Comment:
Please update all published projects to .NET 8:

![image](https://github.com/SergiiKram/workflow-core/assets/2759478/3f06a179-9ad6-4d82-a4af-d24cf3cf08ea)

Reply:
I updated only those projects that target .net6.
WorkflowCore, WorkflowCore.Providers.Redis, WorkflowCore.DSL, WorkflowCore.Testing targets netstandard2.0. I think we don't need to make them framework-specific. What do you think?

Reply:
Ahh, I thought they already target net6. 
So then ok, no need to update them.